_data/en/settings.yml

```diff
@@ -42,20 +43,19 @@ contact_page:
     form_helpers:
       select_topic: '- Select a topic -'
       select_issue: '- Select a specific issue -'
-      select_agency: '- Select an agency application -'
+      select_agency: '- Agency you’re trying to access -'
     labels:
-      agency_application: Agency application
+      agency_application: Partner Agency
       agency_application_all: All Applications
       agency_application_common: Common Applications
       agency_application_other: Other Applications
-      description: Provide more detail about your question
+      description: Provide more detail. If you need support, please describe the steps you took and the issues you encountered.
       email: Email
       first_name: First name
       last_name: Last name
-      pii_warning: Please do **not** include sensitive or identifying information in this section such as your full name, birth date or social security number.
+      pii_warning:  Do not include phone numbers, membership numbers, full name, birthday, address, social security numbers or other identifying information.
```

Comment:
1. supernit, remove the extra leading space?
```suggestion
      pii_warning:  Do not include phone numbers, membership numbers, full name, birthday, address, social security numbers or other identifying information.
```

2. Also there's a mix of plural/singular in the attributes, is that intentional? (ex I'd say full name_s_, birthday_s_, address_es_) toma ke it consistent with phone numbers, membership numbers, social security numbers

Reply:
I believe it is intentional. since those tend to be singular but may be something to ask @Danielle-Lee about as well.

Reply:
@zachmargolis I see your point. I know we wanted to get this pushed before the end of sprint. I may end up updating the language I want to get another opinion you may be right. If it does need to change, would that be possible in another ticket/next sprint? @mdiarra3

Reply:
Gut checked with the UX team and we are making the change. New content added to the word doc and translations submitted. Good flag, @zachmargolis Thank you!